### PR TITLE
Tentative fix for #252

### PR DESCRIPTION
**Pull request by jmcs**

### CODE REVIEW

**Comment by rafaelcaricio** [comment]
Link to #252

File: senza/traffic.py [hunk 1]
```diff
@@ -22,17 +22,20 @@ def get_weights(dns_names: list, identifier: str, all_identifiers) -> ({str: int
...
+        for record in get_records(dns_name.split('.', 1)[1]):
+            if record['Type'] == 'CNAME' and record['Name'] == dns_name:
+                try:
+                    if record['Weight']:
+                        weight = int(record['Weight'])
+                    else:
+                        weight = 0
+                except KeyError:
+                    continue
+                known_record_weights[record['SetIdentifier']] = weight
```
Comment (rafaelcaricio) [inline senza/traffic.py]:
```
NameError: name 'weight' is not defined
```

**Comment by hjacobs** [comment]
:+1:

**Comment by hjacobs** [comment]
No tests?

**Comment by rafaelcaricio** [comment]
👍

**Pull request merged**